### PR TITLE
Feature/fix url

### PR DESCRIPTION
**Pull request by lxpause**
add alternative to get_url

### CODE REVIEW

File: tasks/setup_sjk.yml [hunk 1]
```diff
@@ -14,19 +14,17 @@
...
-        version: "{{ sjk_version }}"
-        repository_url: "{{ sjk_repo_url | default(omit) }}"
-        state: present
-        username: "{{ sjk_repo_username | default(omit) }}"
-        password: "{{ sjk_repo_password | default(omit) }}"
-        timeout: "{{ __sjk_maven_artifact_timeout }}"
-        dest: "{{ sjk_dest }}"
-
+- name: "Download SJK binary"
+  maven_artifact:
```
Comment (lrk) [inline tasks/setup_sjk.yml]: @lxpause `maven_artifact` require lxml, please add:
```
- name: "Ensure system required packages are installed"
  package:
    name: "{{ item }}"
    state: present
  with_items:
    - "python-lxml"
```
at the start of the file

**Comment by lxpause** [comment]
Done

File: defaults/main.yml [hunk 2]
```diff
@@ -2,18 +2,18 @@
...
+sjk_version: latest
 
 # true: use the full sjk binary (with mxdump)
 # false: use lighter sjk binary (without mxdump)
 # Default value: false
 sjk_use_sjkplus: false
 
 # SJK Binary destination path
-sjk_dest: '/opt/sjk/sjk-{{ sjk_version | lower }}'
+sjk_dest: '/opt/sjk/sjk{% if sjk_use_sjkplus is defined and sjk_use_sjkplus== true %}-plus{% endif %}-{{ sjk_version | lower }}'
```
Comment (lrk) [inline defaults/main.yml]: good practice is to use space before and after operators: `sjk_use_sjkplus== true` -> `sjk_use_sjkplus == true`

File: .travis.yml [hunk 3]
```diff
@@ -1,4 +1,6 @@
 ---
+sudo: required
```
Comment (lrk) [inline .travis.yml]: why. sudo. is required ?

File: defaults/main.yml (hunk 2 above)
Comment (lxpause) [reply defaults/main.yml]: sure. done

File: .travis.yml (hunk 3 above)
Comment (lxpause) [reply .travis.yml]: remove that

**Pull request merged**